### PR TITLE
fixes some locker overlays

### PR DESCRIPTION
**Pull request by tf-4**
## About the Pull Request

closes https://github.com/Skyrat-SS13/Skyrat-tg/pull/21389
closes https://github.com/Skyrat-SS13/Skyrat-tg/pull/21409

From that PR,
"https://github.com/Skyrat-SS13/Skyrat-tg/commit/fae04f1fb6674a39cb350d0dfdb422a14a3bba7a made it so that lockers use their base_icon_state instead of icon_state for door overlay if none is explicitly set. As seen below.

image

The lockers on the DS2 are base `/obj/structure/closet/secure_closet` with varedited icon_state so they now need to have their `icon_door` varedited as well to explicitly set the door overlay. Otherwise they will use the grey base secure closet sprites."

image

:cl:
fix: DS-2 lockers will no longer have mismatched doors.
/:cl:

### CODE REVIEW

**Comment by Jolly-66** [comment]
image


my pr is better cause i tested it AND have proof

File: modular_skyrat/modules/Syndie_edits/code/secure_closets.dm [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+	icon_state = "science"
+	icon_door = "science"
+
+/obj/structure/closet/secure_closet/syndicate/scientist
+	name = "scientist gear locker"
+	icon_state = "science"
+	icon_door = "science"
+
+/obj/structure/closet/secure_closet/syndicate/admiral
+	name = "\proper station admiral's locker"
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/Syndie_edits/code/secure_closets.dm]: My only question is, why is it labelled with `\proper`, if `Station Admiral` is just the Syndicate's version of `Captain`, which isn't a proper word?

File: modular_skyrat/modules/Syndie_edits/code/secure_closets.dm [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+/obj/structure/closet/secure_closet/syndicate_leader/armory/munitions
+	name = "armory munitions locker"
+
+/obj/structure/closet/secure_closet/syndicate_leader/brig_officer
+	name = "brig officer gear locker"
+	icon_state = "sec"
+	icon_door = "sec"
+
+/obj/structure/closet/secure_closet/syndicate_leader/corp_liaison
+	name = "\proper corporate liaison's locker"
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/Syndie_edits/code/secure_closets.dm]: Same here.

File: modular_skyrat/modules/Syndie_edits/code/secure_closets.dm [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+	icon_state = "hop"
+	icon_door = "hop"
+
+/obj/structure/closet/secure_closet/syndicate_leader/deck_officer
+	name = "deck officer's locker"
+	icon_state = "qm"
+	icon_door = "qm"
+
+/obj/structure/closet/secure_closet/syndicate_leader/mod
+	name = "MODsuit module locker"
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/Syndie_edits/code/secure_closets.dm]:
```suggestion
	name = "\improper MODsuit module locker"
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Also PR body isn't meeting the PR template, and doesn't show proof of testing.

File: modular_skyrat/modules/Syndie_edits/code/secure_closets.dm (hunk 1 above)
Comment (tf-4) [reply modular_skyrat/modules/Syndie_edits/code/secure_closets.dm]: idk thats just how it was before

**Pull request closed without merging**